packages/ffe-datepicker-react/src/calendar/MonthPicker.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { func, string } from 'prop-types';
+
+export default function MonthPicker(props) {
+    const { previousMonthHandler, nextMonthHandler, month } = props;
+
+    function left(evt) {
+        previousMonthHandler(evt);
+    }
+
+    function right(evt) {
+        nextMonthHandler(evt);
+    }
+
+    return (
+        <div className="ffe-calendar__monthpicker">
+            <div className="ffe-calendar__monthpicker-center">{month}</div>
+            <div
+                role="none"
```

Comment:
vad er poenget med en role="none" her. En div har ju fra starten ingen semantisk mening.

Reply:
Muligens for å omgå linting av at div med onclick ikke er greit?

Reply:
Yes, to pass the linter requirements.

Reply:
The linter is there to help you  ;)